contract-map.json

```diff
@@ -3805,6 +3805,13 @@
     "erc20": true,
     "symbol": "rETH",
 	"decimals": 18
+  },
+  {
+    "name": "Quid Ika",
+    "logo": "quidika.png",
```

Comment:
png as weel as contract address is required

Reply:
@MRabenda updated

Reply:
Ups I made mistake SVG format of image is required to merge